src/atoms/Size/Size.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+
+import Text from '../Text';
+import Size from './Size';
+
+storiesOf('Size', module)
+  .addParameters({
+    component: Size,
+  })
+  .add('default', () => (
+    <Size
+      height={text('height', '100%')}
+      width={text('width', '10px')}
+      maxHeight={text('maxHeight', '20px')}
+      maxWidth={text('maxWidth', '20px')}
+      minHeight={text('minHeight', '5px')}
+      minWidth={text('minWidth', '5px')}
```

Comment:
This story is not helpful visually. Can you try to add a background color or border so we'll know the dimensions when playing around knobs?

Reply:
I have provided a text component for that

Reply:
Yes but it's not possible to visually identify if the container height / width has been changed or not.

Reply:
No matter what height / width I set, the Text is rendered same.

<img src="https://user-images.githubusercontent.com/5961873/77880060-6c314900-7279-11ea-822d-68788df4fb1b.png" height="300px" />

Reply:
should we add  overflow: hidden; ideally child component should occupy the parent space not more than that  @dhruvdutt